src/Tribe/Admin_List.php

```diff
@@ -175,22 +175,28 @@ public static function filter_by_aggregator_record( array $clauses, WP_Query $wp
 			}
 
 			// Check if filtering by aggregator record.
-			$record_num = (int) tribe_get_request_var( 'aggregator_record', 0 );
-			if ( 0 === $record_num ) {
+			$parent_record_id = (int) tribe_get_request_var( 'aggregator_record', 0 );
+			if ( 0 === $parent_record_id ) {
 				return $clauses;
 			}
 
 			global $wpdb;
-			
+
+            		$child_args = array(
+                		'post_type'   => TEC_AR::$post_type,
+                		'post_parent' => $parent_record_id,
+            		);
```

Comment:
Additionally I think you need to set a limit to the number of results here based on the number of events per page. I think `get_option( 'posts_per_page' );` should work.  As I think this one returns all the available results and in some instances can be harmful. 

But need to work with the previous and future pages if any as well.

Reply:
Hmm. That's true, but using 'posts_per_page' wouldn't be accurate. We would want events from all child records of the parent id. Maybe it would be better to alter the SQL statement to use a temp table. Let me work something up... I'll ping you when I have something.

Reply:
I took the get_children() call out of there and handled it via the SQL statement. Not sure why I didn't think of it before.